### PR TITLE
Add recipes for OpenAL support

### PR DESCRIPTION
**Pull request by krinnewitz**
This PR adds recipes for PyOpenAL, OpenAL Soft, PyOgg, libvorbis, and libogg.

Note: A working numpy recipe is needed. So for testing with python3, you also need to pull in #1343.

### CODE REVIEW

File: pythonforandroid/recipes/pyopenal/__init__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from pythonforandroid.recipe import PythonRecipe
+from os.path import join
+
+
+class PyOpenALRecipe(PythonRecipe):
+    version = '0.7.3a1'
+    url = 'https://files.pythonhosted.org/packages/9f/c5/738c76f0f5fad3c4662a1a08108232fcb510f29c367f06029a6c6a2792cf/PyOpenAL-{version}.tar.gz'
```
Comment (AndreMiras) [inline pythonforandroid/recipes/pyopenal/__init__.py]: How about:
```python
url = 'https://files.pythonhosted.org/packages/source/p/pyopenal/PyOpenAL-{version}.tar.gz'
```

File: pythonforandroid/recipes/pyogg/__init__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from pythonforandroid.recipe import PythonRecipe
+from os.path import join
+
+
+class PyOggRecipe(PythonRecipe):
+    version = '0.6.4a1'
+    url = 'https://files.pythonhosted.org/packages/ba/44/ca94dd062fcf6b59488df6052070e0d533a618c2321cdfe991afa6e4d65a/PyOgg-{version}.tar.gz'
```
Comment (AndreMiras) [inline pythonforandroid/recipes/pyogg/__init__.py]:
```python
url = 'https://files.pythonhosted.org/packages/source/p/pyogg/PyOgg-{version}.tar.gz'
```

**Comment by krinnewitz** [comment]
Yes, squashing is fine with me. I will do that in a second.

**Comment by AndreMiras** [comment]
It compiled fine on CrystaX, but didn't try to play with an actual app.
Here's the proof:
```
buildozer android debug
...
[INFO]:    Using Crystax NDK r9c
[INFO]:    Found virtualenv at /home/andre/.local/bin/virtualenv
[INFO]:    Found the following toolchain versions: ['4.9', '5', 'clang3.6', 'clang3.7']
[INFO]:    Picking the latest gcc toolchain, here 5
[INFO]:    Of the existing distributions, the following meet the given requirements:
[INFO]:         pr1348: includes recipes (hostpython3crystax, libogg, libvorbis, openal, sdl2_image, sdl2_mixer, sdl2_ttf, python3crystax, numpy, sdl2, setuptools, six, pyjnius, pyogg, pyopenal, kivy), built for archs (armeabi-v7a)
[INFO]:    pr1348 has compatible recipes, using this one
[INFO]:    Of the existing distributions, the following meet the given requirements:
[INFO]:         pr1348: includes recipes (hostpython3crystax, libogg, libvorbis, openal, sdl2_image, sdl2_mixer, sdl2_ttf, python3crystax, numpy, sdl2, setuptools, six, pyjnius, pyogg, pyopenal, kivy), built for archs (armeabi-v7a)
[INFO]:    pr1348 has compatible recipes, using this one
[INFO]:    -> directory context /home/andre/workspace/test/.buildozer/android/platform/build/dists/pr1348
[INFO]:    Selecting java build tool:
[INFO]:    Detected highest available build tools version to be 28.0.2
[INFO]:        Building with gradle, as gradle executable is present
[DEBUG]:   -> running gradlew assembleDebug
[DEBUG]:        Incremental java compilation is an incubating feature.
[DEBUG]:        :preBuild UP-TO-DATE
[DEBUG]:        :preDebugBuild UP-TO-DATE
[DEBUG]:        :checkDebugManifest
[DEBUG]:        :prepareDebugDependencies
[DEBUG]:        :compileDebugAidl
[DEBUG]:        :compileDebugRenderscript
[DEBUG]:        :generateDebugBuildConfig
[DEBUG]:        :generateDebugResValues
[DEBUG]:        :generateDebugResources
[DEBUG]:        :mergeDebugResources
[DEBUG]:        :processDebugManifest
[DEBUG]:        :processDebugResources
[DEBUG]:        :generateDebugSources
[DEBUG]:        :incrementalDebugJavaCompilationSafeguard
[DEBUG]:        :javaPreCompileDebug
[DEBUG]:        :compileDebugJavaWithJavac
[DEBUG]:        :compileDebugJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
[DEBUG]:        Note: Some input files use or override a deprecated API.
[DEBUG]:        Note: Recompile with -Xlint:deprecation for details.
[DEBUG]:        :compileDebugNdk UP-TO-DATE
[DEBUG]:        :compileDebugSources
[DEBUG]:        :mergeDebugShaders
[DEBUG]:        :compileDebugShaders
[DEBUG]:        :generateDebugAssets
[DEBUG]:        :mergeDebugAssets
[DEBUG]:        :transformClassesWithDexForDebug
[DEBUG]:
[DEBUG]:        Running dex as a separate process.
[DEBUG]:
[DEBUG]:        To run dex in process, the Gradle daemon needs a larger heap.
[DEBUG]:        It currently has 1024 MB.
[DEBUG]:        For faster builds, increase the maximum heap size for the Gradle daemon to at least 1536 MB.
[DEBUG]:        To do this set org.gradle.jvmargs=-Xmx1536M in the project gradle.properties.
[DEBUG]:        For more information see https://docs.gradle.org/current/userguide/build_environment.html
[DEBUG]:
[DEBUG]:        :mergeDebugJniLibFolders
[DEBUG]:        :transformNativeLibsWithMergeJniLibsForDebug
[DEBUG]:        :transformNativeLibsWithStripDebugSymbolForDebug
[DEBUG]:        :processDebugJavaRes UP-TO-DATE
[DEBUG]:        :transformResourcesWithMergeJavaResForDebug
[DEBUG]:        :validateSigningDebug
[DEBUG]:        :packageDebug
[DEBUG]:        :assembleDebug
[DEBUG]:
[DEBUG]:        BUILD SUCCESSFUL
[DEBUG]:
[DEBUG]:        Total time: 3.495 secs
[DEBUG]:
[INFO]:    <- directory context /home/andre/workspace/test/.buildozer/android/platform/python-for-android-new-toolchain
[INFO]:    # Copying APK to current directory
[INFO]:    # APK filename not found in build output, trying to guess
[INFO]:    # Found APK file: /home/andre/workspace/test/.buildozer/android/platform/build/dists/pr1348/build/outputs/apk/pr1348-debug.apk
[INFO]:    # Add version number to APK
[INFO]:    # APK renamed to pr1348-0.1-debug.apk
[DEBUG]:   -> running cp /home/andre/workspace/test/.buildozer/android/platform/build/dists/pr1348/build/outputs/apk/pr1348-debug.apk pr1348-0.1-debug.apk
Could not find hostpython, will not compile to .pyo (this is normal with python3)
No compiled python is present to zip, skipping.
this should only be the case if you are using the CrystaX python
# Android packaging done!
# APK pr1348-0.1-debug.apk available in the bin directory
[andre:~/workspace/test] 1m28s $
```

**Pull request merged**